examples/src/debugger/gizmos.rs

```diff
@@ -28,15 +28,15 @@ pub fn draw_triangles_debug_data_gizmos(
     let Some(snapshot) = debug_data.current_snapshot() else {
         return;
     };
-    let vertices = &debug_data.vertices;
+    let vertices: Vec<Vec3> = debug_data.vertices.iter().map(|v| Vec3::from_array(v.to_array())).collect();
```

Comment:
This conversion is unnecessary since we now use bevy::math type everywhere in the examples
```suggestion
    let vertices = &debug_data.vertices;
```